microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
+Your extension and associated metadata must clearly provide information about the types of in-product purchases offered and the range of prices. You must not mislead users and must be clear about the nature of your in-product promotions and offerings including the scope and terms of any trial experiences.
 
-*   If your extension collects credit card information or uses a third-party payment processor that collects credit card information, the payment processing must meet the current PCI Data Security Standard (PCI DSS).
+If your extension restricts access to user-created content during or after a trial, then you must notify users in every step of the process.
 
-#### 1.8.2 Disclosing paid features
+In addition, your extension must make it clear to users that they are initiating a purchase option in your extension.
 
-Your extension and associated metadata must provide information about the types of in-product purchases offered and the range of prices.  You must not mislead users and must be clear about the nature of your in-product promotions and offerings including the scope and terms of any trial experiences.
 
-If your extension restricts access to user-created content during or after a trial, you must notify users in advance.  In addition, your extension must make it clear to users that they are initiating a purchase option in your extension.
+<!-- ------------------------------ -->
+#### 1.9 Notifications
 
-### 1.9 Notifications
+Your extension must respect system settings for notifications. Any presentation of ads and notifications to users must be consistent with user preferences, regardless of whether the notifications are provided by the Microsoft Push Notification Service (MPNS), Windows Push Notification Service (WNS), or any other service.
 
-Your extension must respect system settings for notifications.  This means any presentation of ads and notifications to users must be consistent with user preferences, regardless of whether the notifications are provided by the Microsoft Push Notification Service (MPNS), Windows Push Notification Service (WNS), or any other service.  If the user disables notifications, either on a product-specific or system-wide basis, your extension must remain functional.
+If the user disables notifications, either on a product-specific or system-wide basis, then your extension must adhere to the user's notification settings and remain functional even though notifications may be disabled.
 
-If your product uses MPNS or WNS to transmit notifications, it must comply with the following requirements:
+You'll be receiving notifications related to your accounts, including but not limited to, login attempts, account changes, and security alerts.  To ensure the security of your accounts and the add-ons you submit, verify your accounts through a secure and approved method.
```

Comment:
Also, you will be receiving notifications

Reply:
added Also,